langworld_db_data/adders/listed_value_adder.py

```diff
@@ -37,15 +37,22 @@ def add_listed_value(
                 new_value_ru=new_value_ru,
                 index_to_assign=index_to_assign,
             )
+        except ValueError as e:
+            raise ListedValueAdderError(
+                f"Failed to add new value to inventory of listed values. {e}"
+            )
 
+        try:
             self._mark_value_as_listed_in_feature_profiles(
                 feature_id=feature_id,
                 new_value_id=id_of_new_value,
                 new_value_ru=new_value_ru,
                 custom_values_to_rename=custom_values_to_rename,
             )
         except ValueError as e:
-            raise ListedValueAdderError(e)
+            raise ListedValueAdderError(
```

Comment:
@MariAnShum maybe create a separate issue for this (add test to fix codecov warning)?

Reply:
@MariAnShum this comment is still valid

Reply:
Done!